envbuilder.go

```diff
@@ -137,6 +137,12 @@ type Options struct {
 	// a Dockerfile is not found.
```

Comment:
Should we update this to mention it also falls back on build failure unless `ExitOnBuildFailure` is set?

Reply:
Good point, will change!

Reply:
Updated!

Reply:
LGTM